### PR TITLE
0 is falsey in JavaScript

### PR DESCRIPTION
**Pull request by matthew-andrews**
so make the max-age check a bit cleverer so… that we don't forget to set it when it's 0

oh javascript.

### CODE REVIEW

File: src/middleware/cache.js [hunk 1]
```diff
@@ -30,7 +30,9 @@ const presets = {
 
 function headerObjToString(obj) {
 	const parts = [];
-	if (obj['max-age']) {
+
+	// HACK: Use someething smarter than just an if (obj['max-age']) { because in JavaScrypt 0 is *falsey*
+	if (obj.hasOwnProperty('max-age')) {
```
Comment (wheresrhys) [inline src/middleware/cache.js]: max-age in obj?

**Comment by wheresrhys** [comment]
:+1:

File: src/middleware/cache.js (hunk 1 above)
Comment (matthew-andrews) [inline src/middleware/cache.js]: i knew there'd be some hipster way of doing this

**Comment by matthew-andrews** [comment]
and again with more hipster^^

**Pull request merged**